[2022-12-08T22:00:01.283+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: back_in_avro.First_task_backup scheduled__2022-12-08T21:00:00+00:00 [queued]>
[2022-12-08T22:00:01.297+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: back_in_avro.First_task_backup scheduled__2022-12-08T21:00:00+00:00 [queued]>
[2022-12-08T22:00:01.298+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-08T22:00:01.299+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2022-12-08T22:00:01.300+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-08T22:00:01.320+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): First_task_backup> on 2022-12-08 21:00:00+00:00
[2022-12-08T22:00:01.329+0000] {standard_task_runner.py:55} INFO - Started process 4086 to run task
[2022-12-08T22:00:01.335+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'back_in_avro', 'First_task_backup', 'scheduled__2022-12-08T21:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/dag_back_in_avro.py', '--cfg-path', '/tmp/tmpxnu_pb67']
[2022-12-08T22:00:01.338+0000] {standard_task_runner.py:83} INFO - Job 25: Subtask First_task_backup
[2022-12-08T22:00:01.429+0000] {task_command.py:389} INFO - Running <TaskInstance: back_in_avro.First_task_backup scheduled__2022-12-08T21:00:00+00:00 [running]> on host c4fdd5f7672d
[2022-12-08T22:00:01.523+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Martin_Jurado
AIRFLOW_CTX_DAG_ID=back_in_avro
AIRFLOW_CTX_TASK_ID=First_task_backup
AIRFLOW_CTX_EXECUTION_DATE=2022-12-08T21:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-08T21:00:00+00:00
[2022-12-08T22:00:01.570+0000] {logging_mixin.py:137} INFO - There is not information in the database
[2022-12-08T22:00:01.572+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-12-08T22:00:01.596+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=back_in_avro, task_id=First_task_backup, execution_date=20221208T210000, start_date=20221208T220001, end_date=20221208T220001
[2022-12-08T22:00:01.668+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-08T22:00:01.707+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
